bin/delayed_job

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+
+require File.expand_path(File.join(File.dirname(__FILE__), '..', 'config', 'environment'))
+require 'delayed/command'
+Delayed::Command.new(ARGV).daemonize
```

Comment:
My guess is that in production we don't want `daemonize`? Like I'd expect we run the process in the foreground to make it easier to manage? Like I'd even expect if we add this to our Procfile we'd keep it in the foreground too.

Reply:
this will probably need some changes, yeah. I looked through the methods/option and couldn't a replacement to run in the foreground.